gson/src/main/java/com/google/gson/FieldNamingPolicy.java

```diff
@@ -75,6 +75,24 @@
     }
   },
 
+  /**
+   * Using this naming policy with Gson will modify the Java Field name from its camel cased
+   * form to a upper case field name where each word is separated by an underscore (_).
+   *
+   * <p>Here's a few examples of the form "Java Field Name" ---&gt; "JSON Field Name":</p>
```

Comment:
Should probably use "are" because it talks about plural "examples":
```suggestion
   * <p>Here are few examples of the form "Java Field Name" ---&gt; "JSON Field Name":</p>
```
Or is this correct English grammar?
(Though it affects the documentation of the other existing policies as well.)

Reply:
Yes, you are right, grammar-wise It should be 'Here are'. But I guess 'Here is' would also work in this case, similar to 'Any changes', not seen here but also commonly used. And for consistency, should we just leave it as is?

Reply:
Thanks for considering these review comments!
Unfortunately, I think I made a grammer mistake for this suggestion by accidentally leaving out the "a", it should probably be
> Here are <ins>a</ins> few examples

(Maybe it is also acceptable without the article "a"?)
Sorry for the inconvenience.

Reply:
No problem! In this case, I think "a few" would be more appropriate as we are emphasizing that there are some examples. Also found some readings on the subject: [this](https://www.learnersdictionary.com/qa/The-Difference-Between-Few-and-A-Few-) and [this](https://www.grammarly.com/blog/few-a-few/). Hopefully, now we've got everything and thank you again!

Reply:
For what it's worth, this native speaker considers that both forms are acceptable. "Here are" is technically correct (which is of course the best kind of correct) but "here's" is probably commoner colloquially because it is easier to pronounce. [This](https://dictionary.cambridge.org/us/grammar/british-grammar/there-is-there-s-and-there-are) is a good reference.

On the other hand, saying "few" instead of "a few" would indeed be wrong.

Reply:
Thanks to both of you for the clarification and the informative links!